specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/locations.json

```diff
@@ -322,14 +364,14 @@
             "$ref": "#/definitions/RegionsCapability"
           }
         },
-        "vmSizes": {
+        "vmsizes": {
```

Comment:
I know this property name is not CamelCase, but in order to fix s360 issues, I need to change this to keep consistent with backend.
Our service will GA a new api version in April, in the new api version, I will fix this issue. Now I can create a work item or github issue to track this.
If it is ok, I hope this can check in. @JeffreyRichter You opinion is important for us, I will follow your suggestion. How do you like?

Reply:
The service must remain backward compatible and so this can't change. The swagger must reflect the service behavior.